### PR TITLE
Make Jenkins public

### PR DESCRIPTION
**Pull request by flamingspaz**
- Add ELB in front of admin node
- Add SG & rules to allow HTTP traffic to admin node

### CODE REVIEW

**Comment by flamingspaz** [comment]
@johngian r? hasn't been applied yet but the plan does as expected.

File: admin.tf [hunk 1]
```diff
@@ -54,3 +64,77 @@ resource "aws_instance" "admin-ec2" {
...
+}
+
+resource "aws_elb" "admin-elb" {
+  name                        = "admin-elb"
+  security_groups             = ["${aws_security_group.admin-elb-sg.id}"]
+  subnets                     = ["${aws_subnet.apps-shared-1c.id}"]
+  listener {
+    instance_port             = 80
+    instance_protocol         = "http"
+    lb_port                   = 80
```
Comment (johngian) [inline admin.tf]: I suggest we redirect to ``https`` instead. I don't really like the idea of using jenkins with plain HTTP.

Comment (johngian) [inline admin.tf]: If this is not supported from ELB let's drop ``http``.

**Comment by johngian** [comment]
r+wc

**Comment by flamingspaz** [comment]
I've removed the HTTP listener and sg, we should be good now.

**Comment by johngian** [comment]
looks ok. r+

**Pull request merged**